### DISCUSSION TITLE
Website bug to report - the website is inoperable

### DISCUSSION THREAD

**Discussion by yoannvial** [vidboda/MobiDetails#27] [General]
Hi,
The home page of the site has display problems via the Firefox and google chrome browsers. The requests "ask mobidetails: " return the following message: Bad Request - The CSRF tokens do not match.

**Comment by vidboda** [comment]
Hi,

this message should only occur when the session is too old. Does this occur when you load a new page from scratch (i.e. open a new tab and provide the URL (or use a bookmark or a search engine or so)?

**Reply by yoannvial** [reply]
No, it doesn't work . There may be a problem on our network

Website home page display
![2022-04-13 140646](https://user-images.githubusercontent.com/61030265/163176639-dfb8fc1b-6eae-49dc-bdeb-49e11f687b63.jpg)

**Comment by vidboda** [comment]
Wow looks like you don't have any styling (CSS). However this is not the CSRF token issue. Your network may be using a proxy that blocks CSS/js files and/or cookies (cookies are used in the CSRF token issue). All computers on your network have the same issue?

**Reply by yoannvial** [reply]
Yes, the PC in the next office has the same problem. Thank you for your answer, I will try to check our proxy.

**Comment by vidboda** [comment]
ok let me know if you or your IT consider that I should check additional settings from my side. I can just confirm that several users are currently using the website successfully.

**Comment by vidboda** [comment]
I was thinking: have to tried cleaning the browser cache (files and cookies) to try to force the download of all the files, including styling, and reset the cookies to avoid the CSRF tken issue?
